tests/cypress/integration/1-Initial-steps/branding.cy.js

```diff
@@ -0,0 +1,43 @@
+// <reference types="Cypress" />
+
+describe( 'Branding', function () {
+	before( () => {
+		cy.clearCustomerData();
+		cy.visit( 'wp-admin/index.php?page=nfd-onboarding' );
+	} );
+
+	// since we are setting brand from plugin container, it will not be set to "newfold"
+	// by default even if mm_brand option is deleted from the database
+	it( 'Has ' + Cypress.env('pluginId') + ' CSS when mm_brand does not exist.', () => {
+		cy.exec( 'npx wp-env run cli wp option delete mm_brand' );
+		cy.reload();
+		cy.get( 'body' ).should( 'have.class', `nfd-brand-${ Cypress.env('pluginId') }` );
```

Comment:
Should we move `Cypress.env('pluginId')` to a utility function `getPluginId`? I see this is used in many places. A function would allow us to handle changes to this in a more efficient way.